pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotControllerHealthCheck.java

```diff
@@ -35,7 +35,7 @@
   ControllerConf controllerConf;
 
   @GET
-  @Path("/pinot-controller/admin")
+  @Path("/")
```

Comment:
This will conflict with LandingPageHandler.

Why change this ? It seems compatible with existing